src/View/Helper/HtmlHelper.php

```diff
@@ -5,6 +5,38 @@ class HtmlHelper extends \Cake\View\Helper\HtmlHelper
 {
     use OptionsAwareTrait;
 
+    /**
+     * A list of allowed styles for badges.
+     *
+     * @var array
+     */
+    public $badgeClasses = [
```

Comment:
This array is same as `OptionsAwareTrait::$buttonClasses` so duplicating it under different name doesn't seem DRY. We can just rename `OptionsAwareTrait::$buttonClasses` to a more generic name list `classList` or something and use that everywhere.